server/controllers/Article.js

```diff
@@ -0,0 +1,217 @@
+import slugCreate from 'slug';
+import models from '../database/models';
+import dotenv from 'dotenv';
+import sequelize from 'sequelize';
+import articleRating from '../utils/services/articleRating'
+
+dotenv.config();
+const { Rating, Article, User } = models;
+
+// const decodedToken
+const ArticleController = class {
+  static createArticle(req, res, next) {
+    const {
+      title, description, body,
+    } = req.body.article;
+  
+    // generating a unique slug
+    const timeStamp = Date.now();
+    const slug = slugCreate(`${title} ${req.body.decoded.username} ${timeStamp}`);
+  
+    const authorId = req.body.decoded.id;
+  
+  console.log(req.body.decoded.id);
+    Article.create({
+      slug, title, description, body, authorId,
+    })
+      .then((article) => {
+        res.status(201).json({
+          article: {
+            id: article.id,
+            title: article.title,
+            description: article.description,
+            body: article.body,
+            slug,
+          },
+          message: 'Article Sucessfully Created',
+        });
+      })
+      .catch(next);
+  };
+  
+  static getArticle(req, res, next) {
+    Article.find({ where: { slug: req.params.slug } })
+      .then((article) => {
+        if (article) {
+          User.find({ where: { id: article.authorId } })
+            .then((user) => {
+              res.status(201).json({
+                article: {
+                  slug: article.slug,
+                  title: article.title,
+                  description: article.description,
+                  body: article.body,
+                  author: user.username,
+                },
+                message: 'Article Found',
+              });
+            })
+            .catch(next);
+        } else {
```

Comment:
Instead of nesting and making use of `else`, you can check for no article, example:
`if (!article) return res.status(404).json('article not found');`